ansys/dpf/core/plotter.py

```diff
@@ -83,11 +83,25 @@ def show_figure(self, **kwargs):
         show_axes = kwargs.pop("show_axes", None)
         if show_axes:
             self._plotter.add_axes()
-        return self._plotter.show()
+        return self._plotter.show(**kwargs)
 
 
 class DpfPlotter:
     def __init__(self, **kwargs):
+        """Create a DpfPlotter object.
+
+        Parameters
+        ----------
+        **kwargs : optional
+            Additional keyword arguments for the plotter. For more information,
+            ee ``help(pyvista.plot)``.
```

Comment:
Check spelling, use intersphinx to link to the pyvista help page.  In your `conf.py` add `    "sphinx.ext.intersphinx"` to `extensions` and then:

```py
# Intersphinx mapping
intersphinx_mapping = {
    "python": ("https://docs.python.org/dev", None),
    "numpy": ("https://numpy.org/devdocs", None),
    "pyvista": ("https://docs.pyvista.org/", None),
}
```
Then, you can use :func:\`pyvista.plot\`

Reply:
I updated the code, that's great. Thank you !